tests/framework-tests/core/js/FluidJSTests.js

```diff
@@ -230,6 +230,29 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         });
     });
 
+    fluid.tests.roundToDecimalTests = [
+        {num: 1.555, scale: 5, expected: 1.555},
```

Comment:
I'm trying to add more tests, but getting failures ;(

Part of the problem stems from the fact that Math.round will round -0.5 to 0 instead of -1. ( https://www.ecma-international.org/ecma-262/5.1/#sec-15.8.2.15 ). I suppose my tests should take that into account?

Another edge case is that 0.014999999999999999, 0.0150000000000000001 and 0.015 are all interpreted as the same number. Perhaps that's not a worth while case to test. Thoughts?